asv_bench/benchmarks/benchmarks.py

```diff
@@ -29,43 +29,72 @@
     random_columns,
     random_booleans,
 )
-from modin.config import NPartitions
+
+try:
+    from modin import config  # noqa: F401
+
+    OLD_MODIN_VERSION = False
+except ImportError:
+    OLD_MODIN_VERSION = True
+
+try:
+    from modin.config import NPartitions
+
+    NPARTITIONS = NPartitions.get()
+except ImportError:
+    if not OLD_MODIN_VERSION:
+        from modin.config import CpuCount
+
+        NPARTITIONS = CpuCount.get()
+    else:
+        import os
+        import multiprocessing
+
+        NPARTITIONS = int(os.environ.get("MODIN_CPUS", multiprocessing.cpu_count()))
 
 try:
     from modin.config import TestDatasetSize, AsvImplementation
 
     ASV_USE_IMPL = AsvImplementation.get()
-    ASV_DATASET_SIZE = TestDatasetSize.get()
+    ASV_DATASET_SIZE = TestDatasetSize.get() or "Small"
 except ImportError:
     # The same benchmarking code can be run for different versions of Modin, so in
     # case of an error importing important variables, we'll just use predefined values
     ASV_USE_IMPL = "modin"
-    ASV_DATASET_SIZE = "Big" if NPartitions.get() >= 32 else "Small"
+    ASV_DATASET_SIZE = "Big" if NPARTITIONS >= 32 else "Small"
```

Comment:
so there is no way to specify benchmark parameters for older version? we could try to get values from the environment variables. Yes, that adds another dependency for `modin.config`, if we change the variable name in `modin.config` we also need to change it here, but probably there could be a more elegant solution to do not create these dependencies.

another concern: why we can't just pin `"Small"` default value for `ASV_DATASET_SIZE`, it would be confusing running benches on different machines, and not understanding why DataSet sizes are different

Reply:
I like the approach to use "Small" value regardless count of partitions.

Reply:
what about getting benchmark parameters directly from environment variables in the case of old versions?

Reply:
added